### DISCUSSION TITLE
Batteries and num_batteries

### DISCUSSION THREAD

**Discussion by NoxArt** [amzn/selling-partner-api-models#4380] [Q&A]
Hi, in our listing that are batteries (not a device with a battery, but the actually product is a battery) we have product type BATTERY and in Edit page on website Number of batteries values are labelled as required, even though it doesn't seem to make much sense because it seems to be for devices that contain batteries? ... anyway originally I put in usually Lithium_Ion and Lithium_Polymer, but these enum values were removed. None of the remaining enum values seems to fit ... and we're getting this
![image](https://github.com/user-attachments/assets/fb1ade64-fb6d-40b9-9fa5-7ef4a83f6f3f)

Should Number of batteries be filled for a Battery listing? If yes then what value should a Lithium battery use?

(if no, I assume i can use Patch -> op=delete on that attribute to remove it)

**Comment by weilinggu** [comment]
@NoxArt Are you using [getDefinitionsProductType](https://developer-docs.amazon.com/sp-api/docs/product-type-definitions-api-v2020-09-01-reference#get-definitions2020-09-01producttypesproducttype) to get requirement for product type BATTERY? From a glance at the BATTERY product type JSON schema(in US marketplace), number of batteries is only required if you fill out information for other attributes such as `batteried_included` or `cell_composition` contains certain value such as "lithium_polymer". So number of batteries might be applicable in your use case. As for why number of batteries is required for battery that contains lithium polymer, you might have to consult seller support/seller central.
[Here](https://youtu.be/fraQT3UhFII?si=UAG9p8Mtwr8prUxY) is a video tutorial on product type JSON schema in case you need it.

Thanks
Weiling
Amazon Selling Partner Services
